sources/packages/backend/apps/db-migrations/src/sql/Announcements/Create-announcements-table.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE sims.announcements (
+    id SERIAL PRIMARY KEY,
+    message_title VARCHAR(100) NOT NULL,
+    message VARCHAR(200) NOT NULL,
+    start_date TIMESTAMP WITH TIME ZONE NOT NULL,
+    end_date TIMESTAMP WITH TIME ZONE NOT NULL,
+    target text [] NOT NULL
```

Comment:
For arrays we have been using VARCHAR() []

Reply:
updated